.env

```diff
@@ -41,6 +41,8 @@ NETWORK_ID=1101
 CONSENSUS_TYPE="istanbul"
 BLOCK_TIME=1
 EPOCH=1000
+IN_MEMORY_DISCOVERY_TABLE=false
```

Comment:
Check with @nambrot , he recommended that we add new flags to `.env.integration` as well.

Reply:
I plan to create a separate PR after today's integration deployment that will include this new flag (as well as the updated commit hashes).